### ISSUE TITLE
[Bug?]: The disabled item within the action menu fails WCAG 1.4.3

### ISSUE THREAD

**Issue by chris-gds** [pluralsight/design-system#2148]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Current behavior 😯

The disabled item within the action menu fails WCAG 1.4.3

<img width="517" alt="pluralsight Design system disabled items within Action menu not meeting contrast checks for text size" src="https://github.com/user-attachments/assets/12735959-a69a-4faf-b3cf-ab1a043a51d7">

### Expected behavior 🤔

I would expect a disabled state to still pass checks so low-vision and blind users can still gain the same information. [Consider Disabled buttons suck](https://axesslab.com/disabled-buttons-suck/ )

### Steps to reproduce 🕹

Steps:

1. Visit https://design-system.pluralsight.com/components/actionmenu
2. Check contrast of disabled item

### Context 🔦

NA

### Your environment 🌎

```shell
NA
```
